.github/workflows/master-push.yml

```diff
@@ -107,3 +76,35 @@ jobs:
         run: |
           echo "${{ secrets.DOCKERHUB_PASSWORD }}" | docker login --username rvdockerhub --password-stdin
           docker image push ${TAG}
+
+          
+  cut-release:
+    name: 'Cut Release'
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Did you forget to add the `needs`  key, or is it just a code movement?

Reply:
It was just a code movement. But you're right that would ensure they don't run before the other. let me add this.